rapids_logger/CMakeLists.txt

```diff
@@ -0,0 +1,195 @@
+# =============================================================================
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+# cmake-lint: disable=R0915
+
+include_guard(GLOBAL)
+
+cmake_minimum_required(VERSION 3.26.4 FATAL_ERROR)
+
+include(../rapids_config.cmake)
+
+include(rapids-cmake)
+include(rapids-cpm)
+
+project(
+  RAPIDS_LOGGER
+  VERSION 0.0.1
+  LANGUAGES CXX)
+
+rapids_cmake_build_type(Release)
+
+rapids_cpm_init()
+
+include(../cmake/thirdparty/get_spdlog.cmake)
```

Comment:
What we do here is related to the previous question I asked about whether we should support different instances of rapids-logger wanting to use spdlog differently (dynamic vs static). If we do want to support that, we may need to put this find logic into the function below and have it respond to user parameters.

Reply:
No longer relevant, for now assuming everything in RAPIDS is continuing to use header-only.